src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private Random random = new Random();
+
+    public String getRandomColor() {
+        Color[] colors = Color.values();
+        int index = random.nextInt(colors.length);
+        return colors[index].name();
+    }
+}

```

Comment:
The class `ColorSupplier` is well implemented and follows the checklist guidelines. However, ensure that you are using this class appropriately elsewhere in your code and that you are not instantiating it multiple times unnecessarily, especially inside loops.